packages/odyssey-storybook/src/contributing/GettingStarted.stories.mdx

```diff
@@ -36,7 +36,7 @@ There are many ways to contribute to the project. Code is just one possible mean
 
 You can use [the official codesandbox.io template][6] to experiment with Odyssey code in your browser right now. Or use the sandbox to share interesting patterns or to reproduce defects that need to be fixed!
 
-[6]: https://codesandbox.io/s/odyssey-react-demo-b7y7i
+[6]: https://codesandbox.io/s/okta-odyssey-react-oiy1iy
```

Comment:
this link includes a sandbox that has variable dependency versions to allow our latest code to be used by default